lib/sycamore/sycamore/data/element.py

```diff
@@ -5,8 +5,8 @@
 
 from PIL import Image
 
-from sycamore.data import BoundingBox
-from sycamore.data import Table
+from sycamore.data.bbox import BoundingBox
+from sycamore.data.table import Table
 
```

Comment:
I was getting circular dependencies (can't import sycamore.data because I'm already importing sycamore.data) with some of the changes I was making so I started to switch to direct imports which reduces the problem.  My guess is some of those changes could be reverted, but I don't see the advantage.  I would like to move away from having so much in __init__ since everything in there is forced to be run even if it's not needed.